netbox_data_import/tests/js/contact_candidate_modal.test.js

```
@@ -0,0 +1,202 @@
+        </div>
+      </form>
+    </div>
+    <script id="ndi-candidate-values-by-row" type="application/json">${JSON.stringify(candidates)}</script>
+    <script id="ndi-contact-suggestions-by-row" type="application/json">${JSON.stringify(contactSuggestions)}</script>
+  `;
+  window.EXISTING_RESOLUTIONS = resolutions;
+  window.TomSelect = TomSelect;
+  for (const id of ["contactCandidateName", "contactCandidateEmail", "contactCandidatePhone"]) {
+    new TomSelect(document.getElementById(id), { create: false });
+  }
+  window.eval(controllerSource);
```

Biome (2.5.6)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)